### PR TITLE
Add Blackbox exporter for probing http api endpoint

### PR DESCRIPTION
**Pull request by rohit-dimagi**
For probing serverup.txt and cert expiry check

### CODE REVIEW

File: group_vars/all.yml [hunk 1]
```diff
@@ -22,5 +22,6 @@ grafana_security:
   admin_user: admin
   admin_password: password
 
-
+probe_url:
```
Comment (snopoke) [inline group_vars/all.yml]: can we rename this to something more expressive like `prometheus_http_probe_urls`

File: roles/prometheus/templates/prometheus.yml.j2 [hunk 2]
```diff
@@ -19,17 +19,26 @@ scrape_configs:
...
-        - /etc/prometheus/file_sd/*.json
+  - job_name: blackbox
+    metrics_path: /probe
+    params:
+      module: [http_prometheus]
+    static_configs:
+      - targets:
+{%- for url in probe_url %}        
+         - {{ url }}
+{% endfor %}
```
Comment (snopoke) [inline roles/prometheus/templates/prometheus.yml.j2:30]: do we need the `relabel_configs` portion of the config: https://github.com/prometheus/blackbox_exporter#prometheus-configuration

File: group_vars/all.yml (hunk 1 above)
Comment (rohit-dimagi) [reply group_vars/all.yml]: renamed.

File: roles/prometheus/templates/prometheus.yml.j2 (hunk 2 above)
Comment (rohit-dimagi) [reply roles/prometheus/templates/prometheus.yml.j2:30]: Yes it was needed otherwise it complains about hostname. Added it.

**Pull request merged**